Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -159,9 +159,8 @@ WK_NO_STATIC_INITIALIZERS_Release_NO_ = $(WK_ERROR_WHEN_LINKING_WITH_STATIC_INIT
 WK_NO_STATIC_INITIALIZERS_Production__ = $(WK_ERROR_WHEN_LINKING_WITH_STATIC_INITIALIZERS);
 WK_NO_STATIC_INITIALIZERS_Production_NO_ = $(WK_ERROR_WHEN_LINKING_WITH_STATIC_INITIALIZERS);
 
-// Workaround for rdar://104837911
 WK_MOVE_OBJC_STUBS_LDFLAGS =
-WK_MOVE_OBJC_STUBS_LDFLAGS[config=Debug] = -Wl,-rename_section,__TEXT,__objc_stubs,__TEXT,__text
+WK_MOVE_OBJC_STUBS_LDFLAGS[config=Debug] = -debug_variant
```

Comment:
This setting name is no longer accurate, would you mind changing it? I’m fine with just folding it into OTHER_LDFLAGS now too since it’s a simple argument.